### ISSUE TITLE
Remove input text field from the left side-bar in error-annotation mode

### ISSUE THREAD

**Issue by elenavolodina** [spraakbanken/swell-editor#77]

**Comment by jroxendal** [comment]
Pausing this, it might be better to keep the input there to make the filtering more obvious.

**Comment by cajosc** [comment]
The input text field is used to enter an identifier for a replacement, e.g. `first_name:male_1`. Maybe remove it and reuse the text field from #86?

**Comment by arildm** [comment]
This could use some discussion.

The text field has two purposes currently: entering a numeric label in anonymization mode, and navigating among labels in both modes (type "ca" and it will highlight "O-CAP", press TAB and it will highlight "M-CASE", then press Enter to add the label). Also, #86 will add free-text comments (in both modes, I assume), which *could* use the same input field.

Navigating among labels could be done even without the text field, but leaving it visible makes it easier.

I'm not sure it is a good idea to remove the input field. Anyway we might want to postpone this after #86.
